### DISCUSSION TITLE
Creating another instance runing on another port using different VPN clients subnet

### DISCUSSION THREAD

**Discussion by sougrinoma-sawadogo** [angristan/openvpn-install#856] [Help]
Hello
I want to get to instances of the OpenVPN service runing on different ports
So that I can have two subnets.
The goal is to be able to isolate the VPN clients by assigning them to the desired sub-networks.

**Comment by cyripton** [comment]
> Hello
> I want to get to instances of the OpenVPN service runing on different ports
> So that I can have two subnets.
> The goal is to be able to isolate the VPN clients by assigning them to the desired sub-networks.

I would love to have that feature too!

**Comment by dovanpho** [comment]
me too :D
